yamlfmt.wrap.sh

```diff
@@ -3,15 +3,45 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-set -o xtrace
 
 source "hack/utils.sh"
 
-while IFS= read -r file; do
+# Function to check if a file matches any ignore pattern
+should_ignore_file() {
```

Comment:
We're crossing the "shouldn't be in shell" bar here IMO - what was the behavior that necessitated this? If the wrap/unwrap is correctly written it should fully undo itself, so it shouldn't really matter if it operates over files that the linters are not touching.

Reply:
i reverted this change, so wrap script is untouched